be/src/http/action/stream_load.cpp

```diff
@@ -558,6 +606,8 @@ Status StreamLoadAction::_process_put(HttpRequest* http_req, StreamLoadContext*
                      << ctx->brief();
         return plan_status;
     }
+    LOG(INFO) << "ThriftDebugString:" << apache::thrift::ThriftDebugString(ctx->put_result);
```

Comment:
debug message use VLOG, and remove std::cout

Reply:
fixed